packages/server-core/src/user/sms/awssns.ts

```diff
@@ -1,27 +1,24 @@
-import AWS from 'aws-sdk'
+import { PublishCommand, SNSClient } from '@aws-sdk/client-sns'
 
 import config from '../../appconfig'
 import logger from '../../ServerLogger'
 
+const snsClient = new SNSClient({ region: config.aws.sms.region })
```

Comment:
I think all of the fields from config.aws.sms need to be used in the configuration.